lib/main_containers.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests in SLES
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use containers::utils qw(loadtest_containers);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  load_container_tests
+);
+
+sub load_container_tests {
+    my $runtime = get_required_var('CONTAINER_RUNTIME');
+    boot_hdd_image;
+
+    # Container Image tests
+    if (get_var('FLAVOR', '') =~ /Container-Image/) {
+        loadtest_containers 'host_configuration';
+        loadtest_containers 'docker_image';
+        loadtest_containers 'podman_image';
```

Comment:
We those tests should not be loaded together.

Reply:
My idea was to run those in 1 job for less overhead in openqa jobs, since it's just 2 tests... but yeah, I can change as it is today and we can discuss later.

Reply:
I'd really change that and don't mix `podman` with `docker` tests.

Reply:
done

Reply:
but for the record, see https://openqa.suse.de/tests/7163594